R/use_standalone.R

```diff
@@ -156,10 +156,22 @@ as_standalone_dest_file <- function(file) {
   gsub("standalone-", "import-standalone-", file)
 }
 
-standalone_header <- function(repo_spec, path) {
+standalone_header <- function(repo_spec, path, ref = NULL, host = NULL) {
+  ref_string <- ref %||% "HEAD"
+  host_string <- host %||% "https://github.com"
+  source_comment <-
+    glue("# Source: {host_string}/{repo_spec}/blob/{ref_string}/{path}")
```

Comment:
It works better in Positron to _not_ surround this with `<>`, i.e. the natural link detection copes better with this form.